src/exes/moc.ml

```diff
@@ -89,6 +89,12 @@ let argspec = [
       String.concat " or " valid_metadata_names ^
       ") as `public` (default is `private`)";
 
+  "--suppress-metadata",
+    Arg.String (fun n -> Flags.(suppress_metadata_names := n :: !suppress_metadata_names)),
```

Comment:
Would it make sense to verify that `n` is in `valid_metadata_names`?

Reply:
This is one of those questions that nobody will ever say no to :-)

Reply:
Yes. And Yes,

Reply:
there is `process_public_metadata_names`  already...

Reply:
Done:  ee533c6